<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryEditorWindow</class>
 <widget class="QMainWindow" name="QueryEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbDatabase"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabQuery">
       <attribute name="title">
        <string>Query editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="teQueryEditor"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabResult">
       <attribute name="title">
        <string>Result set</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tvResultSet">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aExecuteQuery"/>
   <addaction name="separator"/>
   <addaction name="aCommit"/>
   <addaction name="aRollback"/>
  </widget>
  <action name="aCommit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/commit.png</normaloff>:/icons/commit.png</iconset>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
   <property name="toolTip">
    <string>Commit</string>
   </property>
  </action>
  <action name="aRollback">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/rollback.png</normaloff>:/icons/rollback.png</iconset>
   </property>
   <property name="text">
    <string>Rollback</string>
   </property>
   <property name="toolTip">
    <string>Rollback</string>
   </property>
  </action>
  <action name="aExecuteQuery">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Execute SQL</string>
   </property>
   <property name="toolTip">
    <string>Execute SQL</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
